app/models/ncr/work_order.rb

```diff
@@ -171,8 +177,12 @@ def final_approver
       end
     end
 
-    def email_approvers
-      Dispatcher.on_proposal_update(self.proposal, self.modifier)
+    def budget_approvals
+      self.individual_approvals.offset(1)
```

Comment:
I totally hear you, but mind if we create a story for it and take care of it separately? Just don't want to hold up this change any longer.